src/ert/ensemble_evaluator/evaluator.py

```diff
@@ -433,7 +440,17 @@ async def _monitor_and_handle_tasks(self) -> None:
             )
             for task in done:
                 if task_exception := task.exception():
-                    logger.error((f"Exception in evaluator task: {task_exception}"))
+                    exc_traceback = "".join(
+                        traceback.format_exception(
+                            None, task_exception, task_exception.__traceback__
+                        )
+                    )
+                    logger.error(
+                        (
+                            f"Exception in evaluator task {task.get_name()}: {task_exception}\n"
+                            f"Traceback: {exc_traceback}"
+                        )
+                    )
                     raise task_exception
```

Comment:
tre traceback is not printed by this latter `raise`?

Reply:
Unfortunately no. Have been testing it (crashing tasks) and traceback needs to be there.